fs/testdata/empty_dir_sync.ts

```diff
@@ -1,7 +1,7 @@
 import { emptyDirSync } from "../empty_dir.ts";
 
 try {
-  emptyDirSync(Deno.args[0]);
+  emptyDirSync(Deno.args[0]!);
```

Comment:
Similar to the [previous comment](https://github.com/denoland/deno_std/pull/4277/files#r1477861640).